vector/v.category/main.c

```diff
@@ -50,6 +51,25 @@ typedef struct {
     int min[FRTYPES], max[FRTYPES];
 } FREPORT;
 
+enum OutputFormat { PLAIN, SHELL, JSON };
+
+void format_json_fr(FREPORT *freport, int fr_type, char *name,
+                    JSON_Array *array)
+{
+    JSON_Object *object;
+    JSON_Value *value;
+    if (freport->count[fr_type] > 0) {
+        value = json_value_init_object();
+        object = json_object(value);
+        json_object_set_string(object, "type", name);
+        json_object_set_number(object, "field", freport->field);
```

Comment:
I wonder if we should call this "layer" and not "field", all user documentation calls this layer, field is used only internally. This has some historical reasons.